README.rst

```diff
@@ -11,49 +11,169 @@
+        get_data_loaders, ConvNet, train, test)
+
+
+    def train_mnist(config):
+        train_loader, test_loader = get_data_loaders()
+        model = ConvNet()
+        optimizer = optim.SGD(model.parameters(), lr=config["lr"])
+        for i in range(10):
+            train(model, optimizer, train_loader)
+            acc = test(model, test_loader)
+            tune.track.log(mean_accuracy=acc)
+
+
+    analysis = tune.run(
+        train_mnist, config={"lr": tune.grid_search([0.001, 0.01, 0.1])})
+
+    print("Best config: ", analysis.get_best_config(metric="mean_accuracy"))
+
+    # Get a dataframe for analyzing trial results.
+    df = analysis.dataframe()
+
+If TensorBoard is installed, automatically visualize all trial results:
+
+.. code-block:: bash
+
+    tensorboard --logdir ~/ray_results
+
+.. _`Tune`: https://ray.readthedocs.io/en/latest/tune.html
+.. _`Population Based Training (PBT)`: https://ray.readthedocs.io/en/latest/tune-schedulers.html#population-based-training-pbt
+.. _`Vizier's Median Stopping Rule`: https://ray.readthedocs.io/en/latest/tune-schedulers.html#median-stopping-rule
+.. _`HyperBand/ASHA`: https://ray.readthedocs.io/en/latest/tune-schedulers.html#asynchronous-hyperband
+
+RLlib Quick Start
+-----------------
+
+`RLlib`_ is an open-source library for reinforcement learning built on top of Ray that offers both high scalability and a unified API for a variety of applications.
+
+.. code-block:: bash
+
+  pip install tensorflow  # or tensorflow-gpu
+  pip install ray[rllib]  # also recommended: ray[debug]
+
+.. code-block:: python
+
+    import gym
+    from gym.spaces import Discrete, Box
+    from ray import tune
+
+    class SimpleCorridor(gym.Env):
+        def __init__(self, config):
+            self.end_pos = config["corridor_length"]
+            self.cur_pos = 0
+            self.action_space = Discrete(2)
+            self.observation_space = Box(0.0, self.end_pos, shape=(1, ))
 
-.. _`Tune`: http://ray.readthedocs.io/en/latest/tune.html
-.. _`RLlib`: http://ray.readthedocs.io/en/latest/rllib.html
+        def reset(self):
+            self.cur_pos = 0
+            return [self.cur_pos]
 
-Installation
-------------
+        def step(self, action):
+            if action == 0 and self.cur_pos > 0:
+                self.cur_pos -= 1
+            elif action == 1:
+                self.cur_pos += 1
+            done = self.cur_pos >= self.end_pos
+            return [self.cur_pos], 1 if done else 0, done, {}
 
-Ray can be installed on Linux and Mac with ``pip install ray``.
+    tune.run(
+        "PPO",
+        config={
+            "env": SimpleCorridor,
+            "num_workers": 4,
+            "env_config": {"corridor_length": 5}})
 
-To build Ray from source or to install the nightly versions, see the `installation documentation`_.
+.. _`RLlib`: https://ray.readthedocs.io/en/latest/rllib.html
 
-.. _`installation documentation`: http://ray.readthedocs.io/en/latest/installation.html
 
```

Comment:
Do we want these links to the papers? If we keep them, should we add the other relevant papers, e.g., RLlib?

Reply:
We should yeah (currently, they can be more informative than the rest of the documentation!)